en/docs/install-and-setup/configuring-api-manager-for-ob.md

```diff
@@ -79,36 +79,75 @@ database server, and the JDBC driver.
    for access without PSU involvement per day. This enables throttling requests according to the `frequencyPerDay` value provided 
    in accounts initiation request. For a request to be throttled, the TPP should initiate the request.
 
-    ``` toml
-    [open_banking_berlin.consent.freq_per_day]
-    enable = true
-    ```
-    
+    1. To configure the custom throttling policy, enable the `frequencyPerDay` property as follows:
+
+        ``` toml
+        [open_banking_berlin.consent.freq_per_day]
+        enable = true
+        ```
+
+    2. Deploy the throttling policy located in `<WSO2_OB_APIM_BERLIN_TOOLKIT_HOME>/repository/resources/frequencyPerDayPolicy` 
+       as a custom throttling policy in the admin portal of WSO2 Open Banking API Manager. See [API Manager Documentation](https://apim.docs.wso2.com/en/4.0.0/design/rate-limiting/advanced-topics/custom-throttling/) for the instructions.
```

Comment:
Shall we tell to use the given throttle key and the policy before telling to see the APIM link. Let's add one sentence before that.

Reply:
+1

Reply:
Fixed in bfc2d00e04fd6b0d079f3d653be008221b2520dd.